src/c++/perf_analyzer/genai-perf/genai_perf/llm_inputs/llm_inputs.py

```diff
@@ -92,6 +102,12 @@ def create_llm_inputs(
         random.seed(random_seed)
 
         if input_type == PromptSource.DATASET:
+            # (TMA-1990) support VLM input from public dataset
+            if output_format == OutputFormat.OPENAI_VISION:
```

Comment:
There's somewhere in this file that checks supported inputs/outputs. Please use that. We don't want any API-specific logic here or we're going to be introducing the same issues again.

Reply:
Moved to the validate_args method